requirements.txt

```diff
@@ -52,8 +52,8 @@ celery==4.1.1
 certifi==2019.3.9
 cffi==1.12.2
 chardet==3.0.4
-citeproc-py==0.4.0
-citeproc-py-styles==0.1.1
+citeproc-py==0.5.1
+citeproc-py-styles==0.1.3
```

Comment:
Just to verify that we roughly have the same structure/results, can you make a "diff" between the entries that are currently under https://zenodo.org/api/csl/styles and what comes with the updated library?